src/legacy/utils/dhis2Helpers.js

```diff
@@ -10,6 +10,9 @@ function getOrgUnitsRoots() {
         .toPromise();
 }
 
+const getOrgUnitsPaths = ouOrPaths =>
+    ouOrPaths.map(ouOrPath => (typeof ouOrPath === "object" && "path" in ouOrPath ? ouOrPath.path : ouOrPath));
```

Comment:
Same as before, we wouldn't use ``typeof`` or ``in operator`` here.

And, actually, I don't see the need for this helper method. We shouldn't mix string/object in a same array. So I'd rather check where this method is being used and transform to ``string[]`` before concatenating the items.

Reply:
This code was for this task: https://app.clickup.com/t/1jvcwha 

And part of the Dialog replacement means supporting different ways of getting OUs. The old code returns objects, the new code returns paths.